tests/e2e/default-mobile/navigation/more-options-menu.wdio-spec.js

```diff
@@ -60,9 +63,19 @@ describe('More Options Menu - Offline User', () => {
   });
 
   describe('Contact tab', () => {
+
+    beforeEach(async () => {
+      await commonPage.goToBase();
+    });
+
+    after(async () => {
+      await searchPage.clearSearch();
+    });
```

Comment:
```suggestion
```

You don't need this after method to clean the search because after all the tests in this file are executed a new chrome window will be created and it will started clean

Reply:
Confirmed and removed.